modules/ega_download/1.0/config/default.yaml

```diff
@@ -0,0 +1,52 @@
+lcr-modules:
+
+    ega_download:
+
+        inputs:
+          # the input file must contain the following columns:
+            # patient_id: unique id representing individual patient
+            # sample_id: unique sample id for each sample
+            # file_name: name of the file after EGA download.
+              # If you don't have this information, it can be returned
+              # in the output of pyega3 -cf </Path/To/CREDENTIALS_FILE> files EGAD<NUM>
+            # file_format: format of the files to be downloaded. For example: bam, cram, fq.gz, fastq.gz
+            # seq_type: seq type of the sample
+            # genome_build: genome build to which the samples are already aligned or will be aligned
+            # EGAF: file ID from EGA
+            # EGAN: sample ID from EGA
+            # EGAD: dataset ID from EGA
+            # EGAS: study ID from EGA
+            sample_csv: "__UPDATE__" # csv sample table containing columns described above.
+        study_id: "__UPDATE__" # String with cohort name that helps you easily identify the downloaded samples.
```

Comment:
Could this have an example value? It's not clear to me if it should be the EGAS or what would later go into the gambl metadata under "cohort"